src/UI/Scripts/Context.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import {v4 as uuidv4} from "uuid";
 
+const secondsToMS = 1000;
```

Comment:
Ooof this feels so dangerous with Marco changing these values like in the past :P

Reply:
This is required for the eslint rule [no-magic-numbers](https://eslint.org/docs/latest/rules/no-magic-numbers).